BnF ISBN.js

```diff
@@ -0,0 +1,119 @@
+{
+	"translatorID": "f349954c-9957-4b5f-be24-1a8bb52f7fbd",
+	"label": "BnF ISBN",
+	"creator": "Abe Jellinek",
+	"target": "",
+	"minVersion": "4.0",
+	"maxVersion": "",
+	"priority": 100,
```

Comment:
Need to adjust the priority here — Open WorldCat is also 100, so the order is probably undefined.

Reply:
OK, I moved LoC to 97 to make room for BnF at 98.

Reply:
Do we not want WorldCat first?

Reply:
I guess @adam3smith said "between GBV/K10 and Worldcat".

How long does WorldCat take? How does WorldCat quality compare to BnF?

Did you put this before K10 because of the speed?

Reply:
(Sorry, I missed @adam3smith's comment above where he suggested this.)

Reply:
Worldcat quality is bad. Coverage is good, so ideal as a fallback

Reply:
In order to fetch metadata for a list of ISBNs from WorldCat, we have to fetch an HTML page for each one, parse it into a document, extract the ISBN, then make a GET request to an API endpoint and parse the RIS that it returns. I haven't timed anything but the last step, but even just that takes about 680 ms on my end. BnF is definitely a _lot_ faster.

BnF is definitely better quality for French publications. BnF's MARC has creators tagged correctly, while WorldCat tends to make everyone an "author." WorldCat is frequently missing series metadata and the like. Getting BnF's authoritative (UNI)MARC is almost certainly better than getting WorldCat metadata lossily exported to RIS.

Reply:
(edited: extract OCLC ID, not ISBN)

Reply:
OK, sounds good.